lib/assert.js

```diff
@@ -67,13 +55,19 @@ function fail(actual, expected, message, operator, stackStartFunction) {
 }
 
 // EXTENSION! allows for well behaved errors defined elsewhere.
-assert.fail = fail;
+assert.fail = function fail(actual, expected, message, operator) {
```

Comment:
I see no harm in keep this modules style and doing:
```js
function fail() {
...
}
assert.fail = fail;
```
Or am I wrong and there's a name clash?